scripts/distribution/stagenet.Dockerfile

```
@@ -58,10 +65,10 @@ COPY --from=node-dashboard /nginx.conf /etc/nginx/sites-enabled/node-dashboard
 COPY --from=build /build-project/concordium-node/target/release/concordium-node /concordium-node
 COPY --from=build /build-project/concordium-node/target/release/node-collector /node-collector
 COPY --from=build /build-project/start.sh /start.sh
-COPY --from=build /build-project/genesis.dat /genesis.dat
+COPY --from=genesis-data /genesis-data/genesis.dat /genesis.dat
 RUN sha256sum /genesis.dat
 
 COPY ./scripts/distribution/supervisord.conf /etc/supervisor/supervisord.conf
 COPY ./scripts/distribution/concordium.conf /etc/supervisor/conf.d/concordium.conf
-COPY ./scripts/distribution/staging-net-client.sh /staging-net-client.sh
-ENTRYPOINT [ "/staging-net-client.sh" ]
+COPY ./scripts/distribution/stagenet.sh /start.sh
+ENTRYPOINT [ "/start.sh" ]
```

I think this is wrong and is overwriting the `start.sh` script that is copied above, leading to failure.

This should be
```suggestion
ENTRYPOINT [ "/stagenet.sh" ]
```